### DISCUSSION TITLE
Building Photoprism locally

### DISCUSSION THREAD

**Discussion by xplosionmind** [photoprism/photoprism#1160] [Q&A] [answered]
Hello everybody, sorry for bothering with such a trivial question, but my technical skills (specially when it comes to Docker) are very limited.

I have a powerful Ubuntu desktop, and I am looking forward to buying two 10TB Hard Disks and mounting them in RAID 1 to the computer, in order to use it as a photo archive which does not necessarily have to be up, running and online all the time.

My question is this: can I use Photoprism as some sort of photo management app? I already configured the computer in order to SSH into it remotely, and I would love to do the same via WebDAV, but formerly I would like to understand how practical it is to use it as a local pictures gallery / photo management software.

The app is conceived to run on a server, but how effective and useful it is to run locally?  
Furthermore, do you have any best practices / suggestions / warnings about doing so?

Thank you very much.

Best,   
T

**Comment by lastzero** [comment, marked as answer]
We recommend using Docker Compose as described in our docs: https://docs.photoprism.org/getting-started/docker-compose/

It will then run locally. You may also build it from source, but be warned this takes time, it's difficult to keep updated and you'll need to install many dependencies manually.
